apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>gov.cms.bfd</groupId>
+        <artifactId>bfd-pipeline-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>bfd-pipeline-rda-bridge</artifactId>
+    <packaging>jar</packaging>
+
+    <name>RDA Bridge</name>
+    <description>Utility project for creating RDA data</description>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
```

Comment:
why do we need this?

Reply:
It configures the compiler to tell it which version to use, which has a side effect of also configuring the IDE (at least for intellij).  It's sort of habit for me to put it in, I hadn't noticed it's defined in the top level POM, also, the BFD repo has since been updated to JDK11 so this is now out of date.

I will remove this so that it uses the same version as the rest of the repo, JDK11